WpfApp5/WpfApp5/ViewModel/ToysViewModel.cs

```diff
@@ -0,0 +1,103 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.ComponentModel;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Input;
+using WpfApp5.Model;
+
+namespace WpfApp5.ViewModel
+{
+    public class ToysViewModel : INotifyPropertyChanged
```

Comment:
VewModel name should follow name of Control/Window it belongs to.
In that case it should be MainWindowViewModel.

Reply:
-1 to source code quality.